fast_api_als/services/verify_phone_and_email.py

```diff
@@ -38,14 +43,30 @@ async def verify_phone_and_email(email: str, phone_number: str) -> bool:
     phone_valid = False
     data = {}
 
+    logger.info(f'Email validation URL: {email_validation_url}')
```

Comment:
These logs are not needed at the 'info' level

Reply:
I dropped them to debug level.